ktor-client/ktor-client-plugins/ktor-client-content-negotiation/ktor-client-content-negotiation-tests/jvm/src/io/ktor/client/plugins/contentnegotiation/tests/JsonContentNegotiationTest.kt

```diff
@@ -7,7 +7,7 @@ package io.ktor.client.plugins.contentnegotiation.tests
 import io.ktor.http.*
 import io.ktor.serialization.*
 import io.ktor.server.application.*
-import io.ktor.server.plugins.*
+import io.ktor.server.plugins.contentnegotiation.*
```

Comment:
package in tests could be changes as well

Reply:
actually, we may need to unify packaging in tests, since now it's random. we can do it later, because it won't be a breaking change.